news_collector/README.rst

```diff
@@ -0,0 +1,67 @@
+NewsCollector: when and why async more is really worth
+======================================================
+
+Async code is less readeable and more difficult to debug, and requires the
+understanding of new concepts like coroutines and event loops.
+
+Being aware of that, I'm certainly not going to use async code just because
+it's cool or fancy.
+
+Nonetheless, a great effort has being devoted to provide Channels as
+a framework suitable for both synchronous and asynchronous contexts, and
+I have a sincere interest in understanding not just **how**, but rather **why**
+it's worth it and **when** it can provide advantages.
+
```

Comment:
The English here is a little hard to understand, but I can clean that up after merge :)

Reply:
👍 please feel free to adjust it :)